src/software/ai/hl/stp/play/free_kick_play.cpp

```diff
@@ -56,9 +56,9 @@ void FreeKickPlay::getNextTactics(TacticCoroutine::push_type &yield, const World
                                                CreaseDefenderTactic::LeftOrRight::RIGHT),
     };
 
-    Angle min_open_angle_for_shot = Angle::fromDegrees(DynamicParameters->getAIConfig()
+    Angle min_open_angle_for_shot = Angle::fromDegrees(DynamicParameters->getAiConfig()
```

Comment:
are we consistent across the codebase on AI vs Ai?

Reply:
created a ticket here: https://github.com/UBC-Thunderbots/Software/issues/1939